htdocs/fourn/facture/list.php

```diff
@@ -113,6 +112,10 @@
 $search_categ_sup = trim(GETPOST("search_categ_sup", 'int'));
 $search_product_category = GETPOST('search_product_category', 'int');
 
+$search_status = GETPOST('search_status', 'alpha');
+$optioncss = GETPOST('optioncss', 'alpha');
+$object_statut = GETPOST('search_statut', 'alpha');
```

Comment:
Why adding search_statut ?  Seems a duplicate of search_status.
Filter on search_status seems already working.

Reply:
pl